codespell_lib/_codespell.py

```diff
@@ -361,12 +362,45 @@ def parse_options(args):
                         help='print LINES of leading context')
     parser.add_argument('-C', '--context', type=int, metavar='LINES',
                         help='print LINES of surrounding context')
+    parser.add_argument('--config', type=str,
+                        help='path to config file.')
 
     parser.add_argument('files', nargs='*',
                         help='files or directories to check')
 
+    # Parse command line options.
     options = parser.parse_args(list(args))
 
+    # Load config files and look for ``tool:codespell`` options.
+    if (
+        options.config
+        or os.path.exists('setup.cfg')
+        or os.path.exists('.codespellrc')
+    ):
```

Comment:
So this would just be `for path in (options.config, 'setup.cfg', '.codespellrc'): if not path or not os.path.isfile(path): continue ...`